quarkus-integration/quarkus/deployment/src/test/java/ai/timefold/solver/quarkus/TimefoldProcessorImmutablePlanningSolutionTest.java

```diff
@@ -0,0 +1,37 @@
+package ai.timefold.solver.quarkus;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import ai.timefold.solver.core.impl.testdata.constraints.DummyConstraintProvider;
+import ai.timefold.solver.core.impl.testdata.domain.immutable.record.TestdataRecordEntity;
+import ai.timefold.solver.core.impl.testdata.domain.immutable.record.TestdataRecordSolution;
+import ai.timefold.solver.core.impl.testdata.domain.immutable.record.TestdataRecordValue;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.test.QuarkusUnitTest;
+
+class TimefoldProcessorImmutablePlanningSolutionTest {
+
+    @RegisterExtension
+    static final QuarkusUnitTest config = new QuarkusUnitTest()
+            .overrideConfigKey("quarkus.timefold.solver.termination.best-score-limit", "0")
```

Comment:
Is this necessary if we're not actually going to be solving?
Let's not reference things we're not using; it creates problems for when those things need to change, and people start worrying why it's even used here.